### DISCUSSION TITLE
Can codesandbox use pnpm version 8?

### DISCUSSION THREAD

**Discussion by devjiwonchoi** [codesandbox/codesandbox-client#8111] [Q&A]
I have an issue with pnpm version 8, but cannot reproduce the repo since it does not break when using pnpm@7.
Looks like `7.1.0` is the default, can I change it?

**Comment by CompuIves** [comment]
Do you have an example sandbox? We're able to run later versions of pnpm by updating the `.codesandbox/Dockerfile` file.
